src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -55,7 +55,33 @@ function slowRequireResolve(moduleName: string, cwd?: string): string {
     cwd: path.resolve(cwd || process.cwd()),
   });
 
-  return result.stdout.toString().trim();
+  const resolution = result.stdout.toString().trim();
+  if (resolution === "") {
+    throw new Error(`Could not resolve module "${moduleName}"`);
+  }
+  return resolution;
+}
+
+const cachedResolutions: { [id: string]: any } = {};
```

Comment:
should this be `CachedModuleMetadata` not `any`?

Reply:
True.